ciao-controller/client.go

```diff
@@ -267,6 +267,31 @@ func (client *ssntpClient) RestartInstance(instanceID string, nodeID string) err
 	return err
 }
 
+func (client *ssntpClient) AssignFloatingIP(floatingIP payloads.FloatingIP) error {
+	floatingIPCmd := payloads.PublicIPCommand{
+		PublicIP:     floatingIP.AssignFloatingIP.Address,
+		PrivateIP:    floatingIP.AssignFloatingIP.InternalIP,
+		TenantUUID:   floatingIP.AssignFloatingIP.TenantUUID,
+		InstanceUUID: floatingIP.AssignFloatingIP.InstanceUUID,
```

Comment:
with the latest changes, it wont be necessary, we'll request only the TenantUUID as required